kornia/augmentation/augmentation.py

```diff
@@ -107,8 +108,14 @@ def apply_transform(
     ) -> torch.Tensor:
         return hflip(input)
 
+    def inverse_transform(
+        self, input: torch.Tensor, transform: Optional[torch.Tensor] = None,
+        size: Optional[Tuple[int, int]] = None, **kwargs
+    ) -> torch.Tensor:
+        return hflip(input)
```

Comment:
Should we be based on transformation matrix for hlip and vflip? The direct hlip will be much faster.

Reply:
how different is this from what we have already ?

Reply:
No no. It is for the inverse. restore back the flip that has been applied.